### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Create a Spark Scala application _Keyboard: After "Postpone" button there is a focus loss for 2 tabs before going to "Install" button.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3634]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.3.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.4.3

**Repro Steps:**
1.Turn on Screen Reader.
2.Download IntelliJ application from the URL: https://www.jetbrains.com/idea/download/#section=windows .
3.Open IntelliJ IDEA application
4.Tab till Tools-->Azure-->Validate Spark Cluster SSL Certificate-->Disable and press enter
5.Tab till View Menu-->Tool Windows-->Azure Explorer and press enter
6.Right click on the SQL Server big data cluster-->Link SQL Server big data cluster and press enter
7.Enter all the required details to link the cluster
8.Tab till New-->Project and press enter
9.Select Azure Spark/HDInsight > Spark Project with Samples (Scala) and press Next
10.Tab till 'Install' button present in the popup.
11.Verify whether focus order is according to the controls present in the pop-up or not.

**Actual Result:**
After "Postpone" button there is a focus loss for 2 tabs before going to "Install" button.

**Expected Result:**
After "Postpone" button there should be no focus loss before going to "Install" button.

**User Impact:**
Keyboard users will face difficulty in knowing where the focus is pointing if there is a focus visibility loss after "postpone" button. 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_focus pointing to static content in scala plug in check.pptx](https://github.com/microsoft/azure-tools-for-java/files/3656198/Bug_focus.pointing.to.static.content.in.scala.plug.in.check.pptx)

**Comment by t-rufang** [comment]
Note: click tab several times in the `Scala Plugin check` dialog and find focus switch to dialog content,
![image](https://user-images.githubusercontent.com/32627233/65664344-ba1b8900-e06b-11e9-926f-6d966da1fa18.png)

**Comment by t-rufang** [comment]
This `Scala Plugin Check` dialog extends from class `com.intellij.openapi.ui.Messages`, which is supported by IntelliJ team and it's out of the scope of our toolkit.

**Comment by t-rufang** [comment]
- Importance of experience: Minimal. Focus is invisible when user tab once after `Postpone` button.
- Quality of experience: Annoying. Users may get confused when there is focus lose, but they can tab one more time to focus on `Install` button.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
As per discussion with SME, Closing the issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749730
